frontend/src/scenes/pipeline/pipelineDefaultEnabledLogic.tsx

```diff
@@ -0,0 +1,55 @@
+import { connect, kea, path, selectors } from 'kea'
+
+import type { pipelineDefaultEnabledLogicType } from './pipelineDefaultEnabledLogicType'
+import { pipelineTransformationsLogic } from './transformationsLogic'
+
+interface PluginContent {
+    title: string
+    description: string
+}
+type PluginContentMapping = Record<string, PluginContent>
+const pluginContentMapping: PluginContentMapping = {
+    GeoIP: {
+        title: 'Capture location information',
+        description:
+            'Enrich PostHog events and persons with IP location data. This is useful for understanding where your users are coming from. This setting can be found under data pipeline.',
+    },
+}
+
+export interface DefaultEnabledType {
+    title: string
+    description?: string
+    id: number
+    enabled: boolean
+}
+
+export const pipelineDefaultEnabledLogic = kea<pipelineDefaultEnabledLogicType>([
+    path(['scenes', 'pipeline', 'pipelineDefaultEnabledLogic']),
+    connect({
+        values: [pipelineTransformationsLogic, ['plugins', 'pluginConfigs']],
+        actions: [pipelineTransformationsLogic, ['toggleEnabled']],
+    }),
+    selectors({
+        pipelineDefaultEnabled: [
+            (s) => [s.plugins, s.pluginConfigs],
+            (plugins, pluginConfigs): DefaultEnabledType[] => {
+                const defaultEnabledPluginIds = Object.values(plugins)
+                    .filter((plugin) => plugin.name in pluginContentMapping)
+                    .map((plugin) => plugin.id)
+                const defaultEnabledPluginConfigs = Object.values(pluginConfigs).filter((pluginConfig) =>
```

Comment:
there could be two pluginConfigs for GeoIP, I'm just ignoring that problem and assume that our flows never get to that place